pkg/smbios/smbios_linux.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2016-2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package smbios
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+)
+
+var systabPath = "/sys/firmware/efi/systab"
+
+// GetSMBIOSBaseEFI finds the SMBIOS entry point address in the EFI System Table.
+func GetSMBIOSBaseEFI() (int64, int64, error) {
```

Comment:
in the func signature, give vars names to indicate what's what. (base, size int64, err error)

Reply:
Dont know we can do that in go. Thanks. Done.